libpromises/failsafe.cf

```diff
@@ -383,7 +383,7 @@ bundle agent failsafe_cfe_internal_call_update
 
     # On Windows we need cf-execd to call update.cf, otherwise the daemons will
     # not run under the SYSTEM account.
-    !windows::
+    !windows.!skip_policy_on_bootstrap::
```

Comment:
I think this should be `!(windows|skip_policy_on_bootstrap)::` Not windows OR skip policy on bootstrap, the current logic reads not windows and not skip policy on bootstrap

Reply:
Same logic. So a preference of how it reads?

https://csawesome.runestone.academy/runestone/books/published/csawesome/Unit3-If-Statements/topic-3-6-DeMorgan.html#de-morgan-s-laws

```
!(a || b) is equivalent to !a && !b
```

Reply:
bah, ignore me.

yes, i do have a preference for (). I get tripped up easily.